wp-content/themes/reconasia/assets/_scss/pages/post.scss

```diff
@@ -154,4 +154,18 @@
     padding-top: rem(24);
     border-top: 1px solid $color-black-130;
   }
+
+  &-meta__tags {
```

Comment:
This is a situation where I would recommend making the parent `<ul>` a flex container, and then using the gap property to separate the list items. That'll ensure there's no weird extra space depending on how many items are in the list or if they're on top or next to each other.